packages/react/src/components/Tree/TreeTitle.tsx

```diff
@@ -57,15 +57,13 @@ class TreeTitle extends UIComponent<ReactProps<TreeTitleProps>> {
   }
 
   public static defaultProps = {
-    as: 'span',
+    as: 'a',
+    href: '#',
```

Comment:
@priyankar205 please keep { as: 'a' } but remove href. This is similar to our Menu strucutre. I have also tested with screen readers and it works ok